data/models/16x-ESRGAN.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "16xESRGAN",
+    "name": "16x ESRGAN",
```

Comment:
In my opinion, the pretrained models should retain the coupled "16xESRGAN" naming scheme, and other models should just lose the scale (we have it as a tag anyway). Exceptions of course are any models where there's two of the same name with the only difference being scale, in which case I'd say put it in parentheses like Example (4x) and Example (2x)

Most people leave out the scale when discussing model names colloquially anyway. The only reason the scales were in the names of the models is because that info wasn't found anywhere else. Here we have the freedom to just not do that.

But i can be convinced otherwise too, if you don't think that's a good idea